src/types.ts

```diff
@@ -12,3 +14,5 @@ export type Latitude = number;
  * A zoom level
  */
 export type Zoom = number;
+
+export type GeoJSONGeometry = Exclude<Geometry, GeometryCollection>;
```

Comment:
Why are we removing the `GeometryCollection`?

Reply:
it's the only type that doesn't has coordinates
do we want `GeometryCollection`?

Reply:
will be implemented in next PR